lib/inputstreamhelper.py

```diff
@@ -550,8 +551,8 @@ def _install_widevine_x86(self):
 
         downloaded = self._http_download()
         if downloaded:
-            busy_dialog = xbmcgui.DialogBusy()
-            busy_dialog.create()
+            progress_dialog = xbmcgui.DialogProgress()
+            progress_dialog.create()
```

Comment:
`heading` is a required argument for  `xbmcgui.DialogProgress().create()`

```
2019-08-20 20:47:55.981 T:140252882970368   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: Required argument 'heading' (pos 1) not found
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon_plugin.py", line 8, in <module>
                                                run(sys.argv)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon.py", line 254, in run
                                                plugin.run(argv)
                                              File "/storage/.kodi/addons/script.module.routing/lib/routing.py", line 115, in run
                                                self._dispatch(path)
                                              File "/storage/.kodi/addons/script.module.routing/lib/routing.py", line 126, in _dispatch
                                                view_func(**kwargs)
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/addon.py", line 226, in play_id
                                                VRTPlayer(kodi).play(dict(video_id=video_id, publication_id=publication_id))
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/vrtplayer.py", line 287, in play
                                                self._kodi.play(stream, video.get('listitem'))
                                              File "/storage/.kodi/addons/plugin.video.vrt.nu/resources/lib/kodiwrapper.py", line 256, in play
                                                if is_helper.check_inputstream():
                                              File "/storage/.kodi/addons/script.module.inputstreamhelper/lib/inputstreamhelper.py", line 917, in check_inputstream
                                                return self._check_drm()
                                              File "/storage/.kodi/addons/script.module.inputstreamhelper/lib/inputstreamhelper.py", line 871, in _check_drm
                                                return self._install_widevine()
                                              File "/storage/.kodi/addons/script.module.inputstreamhelper/lib/inputstreamhelper.py", line 660, in _install_widevine
                                                return self._install_widevine_x86()
                                              File "/storage/.kodi/addons/script.module.inputstreamhelper/lib/inputstreamhelper.py", line 555, in _install_widevine_x86
                                                progress_dialog.create()
                                              File "/storage/.kodi/addons/script.module.kodi-six/libs/kodi_six/utils.py", line 62, in wrapper
                                                return py2_decode(func(*mod_args, **mod_kwargs))
                                            TypeError: Required argument 'heading' (pos 1) not found
                                            -->End of Python script error report<--
```
https://romanvm.github.io/Kodistubs/_autosummary/xbmcgui.html#xbmcgui.DialogProgress

Reply:
Thanks, fixed.